services/web/server/src/simcore_service_webserver/login/handlers.py

```diff
@@ -117,9 +133,79 @@ async def register(request: web.Request):
     return response
 
 
+async def register_phone(request: web.Request):
+    """
+    Submits phone registration
+    - sends a code
+    - registration is completed requesting to 'phone_confirmation' route with the code received
+    """
```

Comment:
If I get it correctly, this is used when a new account is created?

Reply:
yes